torch_geometric/explain/algorithm/base.py

```diff
@@ -37,6 +43,11 @@ def forward(
             target (torch.Tensor): the target of the model.
             target_index (int or torch.Tensor, optional): The target indices to
                 explain. (default: :obj:`None`)
+            node_index (Union[int, Tensor], optional): the node/edge index
+                to explain. Can be a single index if no batch is provided, or a
+                tuple of indices if a batch is provided. only used if the model
+                task level is :obj:`"node"` or :obj:`"edge"`.
+                (default: :obj:`None`)
             **kwargs (optional): Additional keyword arguments passed to
                 :obj:`model`.
         """
```

Comment:
Regarding the loss function right below this, is there a reason this is in the base explainer instead of just GNNExplainer? Do other explanation methods commonly use a loss function?